scripts/prepareDeploy.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
```

Comment:
Would it make sense to add `set -eo pipefail`  the script fails if a command returns an error?